core/client/examples/simple_balance.rs

```diff
@@ -0,0 +1,28 @@
+use client::rpc_client::RpcClient;
+use client::wallet::{Wallet, BalanceState};
+
+use web3::types::{Address};
+use std::str::FromStr;
+
+#[macro_use]
+extern crate log;
+
+#[tokio::main]
+async fn main() {
+    env_logger::init();
```

Comment:
Info should be the default log level for this example.
```suggestion
    let log_level = std::env::var("RUST_LOG").unwrap_or("info".to_owned());
    std::env::set_var("RUST_LOG", log_level);
    env_logger::init();
```